templates/user/dashboard/issues.tmpl

```diff
@@ -30,7 +30,7 @@
 					<div class="ui divider"></div>
 					<a class="{{if not $.RepoIDs}}ui basic primary button{{end}} repo name item" href="{{$.Link}}?type={{$.ViewType}}&sort={{$.SortType}}&state={{$.State}}&q={{$.Keyword}}">
 						<span class="text truncate">All</span>
-						<div class="ui {{if $.IsShowClosed}}red{{else}}green{{end}} label">{{CountFmt .TotalIssueCount}}</div>
+						<span class="ui ">{{CountFmt .TotalIssueCount}}</span>
```

Comment:
```suggestion
						<span class="ui">{{CountFmt .TotalIssueCount}}</span>
```

Reply:
@fsiddi Could you follow the suggestion? Looks like we cannot help you merge directly.